### ISSUE TITLE
Adding new devices Smart Scan Hit or Miss

### ISSUE THREAD

**Issue by heffneil** [AlCalzone/node-zwave-js#7192]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

zwave-js-ui: 9.18.0.5040e7e 

### ZwaveJS version

13.2.0

### Describe the bug

I can not reliably add devices.  I just added a device and it worked. Then I follow the same procedure and it just never even tries to find it.  I have tried 4 or 5 new switches 10 times each.

### To Reproduce

On my Iphone with the latest updates (ios18) I go to provision and smart scan and scan the qr code on these zooz on/off switches. ZEN76 800LR

### Expected behavior

these switches should just get added.  I don't see inclusion mode activate on the top bar on my phone.  I have no idea what is going on but I am losing my mind trying to do this SO many times with poor to no results. Please help

**Comment by robertsLando** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by heffneil** [comment]
Can I paste it here safely?  Also I have another problem where I can't get to my zwave interface no chrome or safari on my phone unless I use an incognito window.  I tried clearing all cookies and browsing data for all time on chrome and I get "Load Failed with RETRY refresh" on the screen.  I have the bottom bar of the interface but that's it.  SO strange.

**Comment by robertsLando** [comment]
> Can I paste it here safely

Yes

> Also I have another problem where I can't get to my zwave interface no chrome or safari on my phone unless I use an incognito window

@heffneil are you using a proxy? Could you show me chrome console?

**Comment by heffneil** [comment]
I am using a cluster of hosts behind nomad with traefik.  I don't know if that clarifies your question but like I said incognito and browsers on the computers work.  Chrome and Safari not so much.

**Comment by heffneil** [comment]
[zwave-js-ui-store.zip](https://github.com/user-attachments/files/17100488/zwave-js-ui-store.zip)

**Comment by robertsLando** [comment]
@heffneil Did you also setup an external auth on your proxy?

About the main issue could you explain me if the qr scan completes sucessfully (so you see the provisioning entry on the table) and then the device is never added or if it's the scan that is failing?

**Comment by heffneil** [comment]
I don't know I am only using this locally.  Smart Scan will just wait and wait for hours and then finally they will come online but it takes far too long.  I added devices manually but it still doesn't get in to the provisioned list.  Just in the smart scan list.

**Comment by robertsLando** [comment]
@heffneil do you scan deviced from "Provisioning Entries" tab or from the control panel nodes manager dialog?

**Comment by heffneil** [comment]
I go in to Provisioning Entities and do smart scan and scan the qr code.  from that point it isn't clear what is happening next but usually it takes HOURS before something will kick in.

**Comment by robertsLando** [comment]
@AlCalzone

**Comment by AlCalzone** [comment]
According to your log, a device starts requesting to join the network via SmartStart at 04:09:48 UTC and continues to do so until 11:34:25 UTC. However the DSK that belongs to those requests is not in your provisioning list, so it does not get included.
I assume that's when you gave up and included it manually.

If you can share one of the QR codes of a device that would not include, I can see if I figure out why. I assume the QR code is incorrect though (swapped boxes maybe?)

**Comment by heffneil** [comment]
I can provide whatever.  I have tried to add tons of devices and when I do the smart scan it can literally take hours.  Now I don't know if i have to put the device in inclusion mode or whatever - but I never do and eventually they seem to be found.  Im just at a loss.  Im also trying to add these devices as long range and despite having the proper configuration with security keys and setting USA Long Range that doesn't seem viable.

@robertsLando I still dont' have resolution on the iPhone unable to render without incognito mode.

**Comment by AlCalzone** [comment]
> if i have to put the device in inclusion mode or whatever - but I never do and eventually they seem to be found

Seems like you're doing it right then. Let me see the QR code from the attempt in your log above.

**Comment by heffneil** [comment]
[zwave-js-ui-store (1).zip](https://github.com/user-attachments/files/17140885/zwave-js-ui-store.1.zip)
More problems today.

**Comment by heffneil** [comment]
I do think I had some success with scanning the QR code, checking the active box and then triple clicking my zooz switch.  But after three it just dragged to a halt - I had to restart the container and then later they started to come in.  There are two that never made it in for some reason.  They show in the list as unknown and I can't seem to get them to get interviewed.

**Comment by AlCalzone** [comment]
Which Z-Wave stick do you have? On which firmware?

**Comment by heffneil** [comment]
I am using a RaZberry 7 Pro with an external antenna on a network adapter from Tube

**Comment by AlCalzone** [comment]
FWIW, dropping a 900000 lines logfile with the comment "more problems" is not particularly specific.
> They show in the list as unknown

Which nodes are "they"?

In any case, that logfile shows that you receive a meter report on average every 2.5 seconds, in some cases much more frequently. If routing is involved, there's likely even more permanent traffic on the mesh.
It's very likely that this has a lot to do with the issues you're seeing. I suggest giving https://zwave-js.github.io/node-zwave-js/#/troubleshooting/first-steps?id=reduce-reports-from-the-network a good read to improve that situation.

**Comment by heffneil** [comment]
I will be honest I didn't look at the log files - I just zipped them up.  I tried to add several devices and played the join the network crapshoot game.  I was able to have 3 devices join quickly and properly and several others that took standing on one leg and patting the top of my head and rubbing my belly to get to join and others that just totally refused.  Now on day 2 I was able to get the last of them joined.  I appreciate the link.  I am struggling with over-saturation of my wifi network and transitioning to zwave devices to clear up the airwaves.  Some growing pains getting the devices in but once they join and I stop adding more devices I think things are fairly reliable.

**Comment by heffneil** [comment]
I see what you are saying and the problem is I looked on the zooz website about reporting and I don't know how to set it report less often.

Parameter 16: Choose how the switch reports its status back to the hub.

Values: 0 – Binary switch report for Z-Wave control, basic set report for paddle control; 1 – binary switch report for both Z-Wave and paddle control (default)

Size: 1 byte dec

**Comment by AlCalzone** [comment]
> I will be honest I didn't look at the log files

What I meant to say, I need specifics to look for in such a huge logfile, e.g. _"Node xyz shows in the list as unknown and I can't seem to get it interviewed."_

> I don't know how to set it report less often.

Which specific device is _it_? Does it do energy metering? Do you have a link to the website?

**Comment by heffneil** [comment]
I do have some lamp modules and other power monitoring devices but truthfully that's not what I am using them for.  99% are zooz devices ZEN04 800LR and 2 of the minoston  MP21Z / MP31Z

**Comment by heffneil** [comment]
I have two nodes that are completely unknown to me.  They have been spinning for over a week now on the control panel page.  they do not appear in the smart start.  I have tried deleting failed nodes.  It still won't go away... Any thought or ideas?

**Comment by heffneil** [comment]
Under Interviewee it says ProtocolInfo and its spinning - that's what i mean above. Restarting the container didn't fix it.

**Comment by AlCalzone** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs) of you trying to remove the failed devices, loglevel `debug` and attach it here as a file (drag & drop into the text field).
Make sure to mention the node IDs, so I know what to look for.

**Comment by heffneil** [comment]
I ran a rebuild routes just after sending this and 10 hours later 90% of them remain.  I am looking to invest in more Z Wave switches but I don't think I can consider it now.  Im just not sure what is wrong in my network.  I headed your advice and disable almost all power reporting except from 2 devices and when I actually can do stuff again I will disable it on them too. any and all advice is GREATLY appreciated.

**Comment by heffneil** [comment]
This is what debug is telling me:
```
2024-10-08T11:47:33.017Z DRIVER     no handlers registered!
2024-10-08T11:47:33.018Z DRIVER     no handlers registered!
2024-10-08T11:47:33.034Z DRIVER     no handlers registered!
2024-10-08T11:47:33.035Z DRIVER     no handlers registered!
2024-10-08T11:47:33.035Z DRIVER     no handlers registered!
2024-10-08T11:47:33.035Z DRIVER     no handlers registered!
2024-10-08T11:47:33.260Z CNTRLR   Serial API restarted unexpectedly.
2024-10-08T11:47:33.260Z CNTRLR   Currently active command will be retried...
2024-10-08T11:47:33.261Z CNTRLR   Starting hardware watchdog...
2024-10-08T11:47:34.608Z DRIVER     no handlers registered!
2024-10-08T11:47:34.608Z DRIVER     no handlers registered!
2024-10-08T11:47:34.609Z DRIVER     no handlers registered!
2024-10-08T11:47:34.782Z DRIVER     no handlers registered!
2024-10-08T11:47:34.783Z DRIVER     no handlers registered!
2024-10-08T11:47:34.783Z DRIVER     no handlers registered!
2024-10-08T11:47:34.783Z DRIVER     no handlers registered!
2024-10-08T11:47:34.800Z DRIVER     no handlers registered!
2024-10-08T11:47:34.800Z DRIVER     no handlers registered!
2024-10-08T11:47:35.026Z CNTRLR   Serial API restarted unexpectedly.
2024-10-08T11:47:35.026Z CNTRLR   Currently active command will be retried...
2024-10-08T11:47:35.027Z CNTRLR   Starting hardware watchdog...
2024-10-08T11:47:36.342Z DRIVER     no handlers registered!
2024-10-08T11:47:36.343Z DRIVER     no handlers registered!
2024-10-08T11:47:36.343Z DRIVER     no handlers registered!
2024-10-08T11:47:36.482Z DRIVER     no handlers registered!
2024-10-08T11:47:36.483Z DRIVER     no handlers registered!
2024-10-08T11:47:36.498Z DRIVER     no handlers registered!
2024-10-08T11:47:36.499Z DRIVER     no handlers registered!
2024-10-08T11:47:36.500Z DRIVER     no handlers registered!
2024-10-08T11:47:36.516Z DRIVER     no handlers registered!
2024-10-08T11:47:36.517Z DRIVER     no handlers registered!
2024-10-08T11:47:36.533Z DRIVER     no handlers registered!
2024-10-08T11:47:36.534Z DRIVER     no handlers registered!
2024-10-08T11:47:36.535Z DRIVER     no handlers registered!
2024-10-08T11:47:36.535Z DRIVER     no handlers registered!
2024-10-08T11:47:36.759Z CNTRLR   Serial API restarted unexpectedly.
2024-10-08T11:47:36.760Z CNTRLR   Currently active command will be retried...
2024-10-08T11:47:36.760Z CNTRLR   Starting hardware watchdog...
2024-10-08T11:47:38.153Z DRIVER     no handlers registered!
2024-10-08T11:47:38.161Z DRIVER     no handlers registered!
2024-10-08T11:47:38.162Z DRIVER     no handlers registered!
2024-10-08T11:47:38.309Z DRIVER     no handlers registered!
2024-10-08T11:47:38.310Z DRIVER     no handlers registered!
2024-10-08T11:47:38.310Z DRIVER     no handlers registered!
2024-10-08T11:47:38.326Z DRIVER     no handlers registered!
2024-10-08T11:47:38.343Z DRIVER     no handlers registered!
2024-10-08T11:47:38.344Z DRIVER     no handlers registered!
2024-10-08T11:47:38.344Z DRIVER     no handlers registered!
2024-10-08T11:47:38.344Z DRIVER     no handlers registered!
2024-10-08T11:47:38.360Z DRIVER     no handlers registered!
2024-10-08T11:47:38.361Z DRIVER     no handlers registered!
2024-10-08T11:47:38.570Z CNTRLR   Serial API restarted unexpectedly.
2024-10-08T11:47:38.570Z CNTRLR   Currently active command will be retried...
2024-10-08T11:47:38.571Z CNTRLR   Starting hardware watchdog...
2024-10-08T11:47:39.819Z DRIVER     no handlers registered!
2024-10-08T11:47:39.837Z DRIVER     no handlers registered!
2024-10-08T11:47:39.838Z DRIVER     no handlers registered!
2024-10-08T11:47:39.994Z DRIVER     no handlers registered!
2024-10-08T11:47:39.995Z DRIVER     no handlers registered!
2024-10-08T11:47:39.995Z DRIVER     no handlers registered!
2024-10-08T11:47:40.012Z DRIVER     no handlers registered!
2024-10-08T11:47:40.013Z DRIVER     no handlers registered!
2024-10-08T11:47:40.013Z DRIVER     no handlers registered!
2024-10-08T11:47:40.013Z DRIVER     no handlers registered!
2024-10-08T11:47:40.237Z CNTRLR   Serial API restarted unexpectedly.
2024-10-08T11:47:40.237Z CNTRLR   Currently active command will be retried...
2024-10-08T11:47:40.238Z CNTRLR   Starting hardware watchdog...
2024-10-08T11:47:41.615Z DRIVER     no handlers registered!
2024-10-08T11:47:41.632Z DRIVER     no handlers registered!
```

**Comment by heffneil** [comment]
```
2024-10-08T12:07:14.820Z CNTRLR   Serial API restarted unexpectedly.
2024-10-08T12:07:14.820Z CNTRLR   Currently active command will be retried...
2024-10-08T12:07:14.821Z CNTRLR   Starting hardware watchdog...
2024-10-08T12:07:15.079Z DRIVER   Dropping message with invalid payload
2024-10-08T12:07:15.110Z CNTRLR   Failed to execute controller command after 1/3 attempts. Scheduling next try i
                                  n 100 ms.
2024-10-08T12:07:15.242Z CNTRLR   Failed to execute controller command after 2/3 attempts. Scheduling next try i
                                  n 1100 ms.
2024-10-08 08:07:15.245 INFO Z-WAVE: [Node 008] Metadata updated: 50-0-value-66049
2024-10-08 08:07:15.248 INFO Z-WAVE: [Node 008] Value updated: 50-0-value-66049 0 => 0
2024-10-08T12:07:15.274Z DRIVER   Dropping message with invalid payload
2024-10-08T12:07:15.450Z DRIVER   Dropping message with invalid payload
2024-10-08T12:07:15.515Z DRIVER   Dropping message with invalid payload```

**Comment by AlCalzone** [comment]
@robertsLando does Z-Wave JS UI have an option to disable the watchdog yet?

**Comment by robertsLando** [comment]
nope, is that a driver option?

**Comment by AlCalzone** [comment]
Yes, added in v12.12.0:
https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md#12120-2024-06-24

Looks like @heffneil needs to use that to fix their issues.

**Comment by robertsLando** [comment]
Seems he can also use the env var `ZWAVEJS_DISABLE_WATCHDOG` for that, what do you think?

**Comment by AlCalzone** [comment]
Depends on where Z-UI runs, but we should have a setting like disabling controller recovery.

**Comment by robertsLando** [comment]
Created an issue on my side, in the meanwhile I think he can use the env var as he said he is using docker

**Comment by heffneil** [comment]
Can I ask why I need controller recovery?

**Comment by heffneil** [comment]
I am using a special configuration of Docker but I see I am running:

zwave-js-ui: 9.18.0.5040e7e
zwave-js: 13.2.0
home id: 4163067376
home hex: 0xf8235df0

**Comment by AlCalzone** [comment]
> Can I ask why I need controller recovery?

You don't. You need to disable the option for Z-Wave JS that enables the hardware watchdog automatically, since your controller seems to constantly restart in a loop when it's active:
https://github.com/zwave-js/node-zwave-js/issues/7192#issuecomment-2399624839

So if you're running Docker directly, try setting the environment variable `ZWAVEJS_DISABLE_WATCHDOG` to something that's not empty, e.g. `1`

**Comment by robertsLando** [comment]
Option will be available on ui starting from next version 9.24.0
